src/implementations/twig/components/banner/banner.html.twig

```diff
@@ -43,8 +83,24 @@
   {% set _css_class = _css_class ~ ' ecl-banner--' ~ _size %}
 {% endif %}
 
-{% if _centered %}
-  {% set _css_class = _css_class ~ ' ecl-banner--centered' %}
+{% if _font_size != 'm' %}
+  {% set _css_class = _css_class ~ ' ecl-banner--font-' ~ _font_size %}
```

Comment:
but..if there are only two options available and we only set this class in case it's not 'm'; isn't it better to harcdode the "l", to avoid potential randomness..?